analyses/cell-type-consensus/references/panglao-cell-type-ontologies.tsv

```diff
@@ -85,27 +85,27 @@ CL:0000893	thymocyte	Thymocytes
 CL:4023169	trigeminal neuron	Trigeminal neurons
 CL:0000351	trophoblast cell	Trophoblast cells
 CL:0000731	urothelial cell	Urothelial cells
-NA	NA	Adipocyte progenitor cells
-NA	NA	Airway epithelial cells
-NA	NA	Airway goblet cells
-NA	NA	Airway smooth muscle cells
-NA	NA	Alpha cells
-NA	NA	Anterior pituitary gland cells
-NA	NA	B cells memory
-NA	NA	B cells naive
-NA	NA	Bergmann glia
-NA	NA	Beta cells
-NA	NA	Cajal-Retzius cells
-NA	NA	Cardiac stem and precursor cells
-NA	NA	Cardiomyocytes
-NA	NA	Choroid plexus cells
-NA	NA	Clara cells
-NA	NA	Crypt cells
-NA	NA	Delta cells
-NA	NA	Distal tubule cells
-NA	NA	Ductal cells
-NA	NA	Endothelial cells (aorta)
-NA	NA	Endothelial cells (blood brain barrier)
+CL:0002334	preadipocyte	Adipocyte progenitor cells
+CL:0002368	respiratory epithelial cell	Airway epithelial cells
+CL:0002370	respiratory goblet cell	Airway goblet cells
+CL:0019019	tracheobronchial smooth muscle cell	Airway smooth muscle cells
+CL:0000171	pancreatic A cell	Alpha cells
+CL:0000637	chromophil cell of anterior pituitary gland	Anterior pituitary gland cells
```

Comment:
I don't think we have enough information to specifically call this a chromophil cell, so I'd recommend going with `CL:2000004` pituitary gland cell http://purl.obolibrary.org/obo/CL_2000004

Reply:
This makes sense to me. Updated in https://github.com/AlexsLemonade/OpenScPCA-analysis/pull/914/commits/e7b4490045d40a184df0e85cfe8434863f1b3522